docs/en/docs/ghippo/user-guide/personal-center/accesstoken.md

```diff
@@ -6,11 +11,18 @@ The access key can be used to access the openAPI and continuous delivery. Users
 
 Log in to DCE 5.0, find __Personal Center__ in the drop-down menu in the upper right corner, and you can manage the access key of the account on the __Access Key__ page.
```

Comment:
Log in to DCE 5.0, find __Personal Center__ in the drop-down menu in the upper right corner, and you can manage the access key of the account on the __Access Keys__ page.